e2e/cypress/integration/system_console/user_management/users_deactivation_spec.js

```diff
@@ -161,4 +170,46 @@ describe('System Console > User Management > Deactivation', () => {
                 find('.icon-archive-outline').should('be.visible');
         });
     });
+
+    it('MM-T947 When deactivating users in the System Console, email address should not disappear', () => {
+        // # Visit the system console.
+        cy.visit('/admin_console');
+
+        // # Go to User management / Users tab
+        cy.findByTestId('user_management.system_users').should('be.visible').click();
+
+        // # Create a new user
+        cy.apiCreateUser().then(({user: user1}) => {
+            // # Search the newly created user in the search box
+            cy.findByPlaceholderText('Search users').should('be.visible').clear().type(user1.email);
```

Comment:
Please add a delay here to account for async DOM refresh. The test is failing for me locally without this delay.

```suggestion
            cy.findByPlaceholderText('Search users').should('be.visible').clear().type(user1.email).wait(TIMEOUTS.HALF_SEC);
```